app/controllers/admin/divisions_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Admin::DivisionsController < ApplicationController
+  before_action :authenticate_user!
+  after_action :verify_authorized
+
+  def select
+    redisplay_url = params[:redisplay_url] || root_path
+    division_id = params[:division_id]
+    set_selected_division_id(division_id)
+    division = Division.find_safe(division_id)
+    authorize division || current_division
```

Comment:
The previous code was always just authorizing against Division.root.  I was trying to mirror that behavior but with with an accessible division.  Would it be better to just bypass the need for an 'authorize' call in this case?  I wasn't sure if that was preferred or off hand how to do that.